client/containers/Pub/PubSyncManager.tsx

```diff
@@ -32,7 +32,7 @@ const shimPubContextProps = {
 	historyData: {},
 	firebaseDraftRef: null,
 	updateLocalData: null as any,
-	updatePubData: null as any,
+	updatePubData: () => ({} as any),
```

Comment:
What you really want here is `(() => {}) as any` but it's not a big deal because I'm about to delete this whole file!